### DISCUSSION TITLE
Relative path images not loading server side

### DISCUSSION THREAD

**Discussion by kaplantm** [vercel/next.js#12626] [Help]
Hello, I'm new to next and trying to display an image and make sure it can render server side.

My image is location:
```
public/images/pick2.png
```

In my page: 
```
<img src="/images/pick2.png" alt="placeholder" />
```

This displays fine in the browser, but when I look at the network tab I see that it did not load server side:
<img width="133" alt="Screen Shot 2020-05-08 at 12 51 45 AM" src="https://user-images.githubusercontent.com/13424028/81371547-27919c80-90c6-11ea-8525-56f0cc47753a.png">

However, it works correctly, rendering both client and server side, if I specify the full path:
```
<img src="http://localhost:3000/images/pick2.png" alt="placeholder" />
```

Is there a way to avoid specifying the full path and still have the images load server side?

**Comment by zackdotcomputer** [comment]
That's odd! What you have in your JSX _should_ work fine with `src="/images/pick2.png"`. My two first-thought questions are: 1. What does your `next.config.js` look like? and 2. How are you running the server (with what command, in what version of node+npm/yarn)?

**Reply by kaplantm** [reply]
I'm running node v12.16.3, npm 6.14.4, and I do not have a next.config.js, I am running the "Initial commit from Create Next App"

**Reply by zackdotcomputer** [reply]
Could you share a github repository where this is happening? Doesn't have to be the one you're working on, if you can spin up a minimal one that has the same issue. I suspect I can help you track down what's wrong but it's hard to do so without seeing the code.

**Reply by kaplantm** [reply]
https://github.com/kaplantm/Nextjs-Image-Test
When i look in the network tab, i see the relative image is not loaded by the exact image is. This is evident with npm run dev and npm run start.

**Reply by zackdotcomputer** [reply]
I cloned that repository and ran exactly the following commands:
```bash
npm install # installs the dependencies from package.json
npm run dev # runs next in dev mode
```

After that, when I visited localhost:3000, I saw both images.

So, I'm now confident that this issue is affecting only your machine and isn't something wrong with the code you're using. The only explanations I can think of for why this isn't working on your machines are (in order of likeliness):

**1. You're installing the dependencies or running the server in a non-standard way.**
How are you starting your server? Are you just running `npm run dev` in the code repository's main directory?

**2. You have an unusual setting on your web browser.**
Can you try this on another web browser on your machine? Maybe in private browsing mode as well? Can you share a screenshot of the network tab with the call to the relative path and the failure?

**3. You have a firewall of some sort that is blocking or capturing this request but not others.**
Are you running something like LittleSnitch or a Firewall or some other security software that might be sending these requests into a blackhole? If you're on a unix-like system, have you edited your `/etc/hosts` file to include a weird mapping for localhost-like values? If you get here and you haven't found it yet, then I'm afraid this is likely something that someone can't help you diagnose over an internet forum :(

**Reply by kaplantm** [reply]
Thanks for the response, I appreciate your time on this. To clarify, I am seeing the images render in browser, so the request is successful. However, when I open the network tab in chrome I can see the exact path image in the request preview, while the relative path image does not appear. I thought this might be an issue with local paths not working correctly with some server side loading thing (very new to SSR) but I realize now that I might be misunderstanding what that tab is actually showing, as I realize I can actually see client-side network requests happening for both images. I'm not sure why the exact/relative paths cause differences in the request preview, but nonetheless they seem to be working the same and this was a misunderstanding on my part. Thanks for your help!

**Reply by zackdotcomputer** [reply]
AAAAAAAAAAH Now I understand I think I can write an answer...

**Comment by zackdotcomputer** [comment]
I think what is happening is that your network tab is collapsing the relative path and absolute path images into one request - it's able to recognize that they are pointing to the same image and so it doesn't make two calls.

**Comment by Adylitto** [comment]
https://github.com/Adylitto/DAprOd.org/tree/main/client
same pb for me cant dispay 2 profile pictures in home.jsx tried many relative. paths still no display on vercel prod
